ci/lint.yml

```
@@ -0,0 +1,15 @@
+---
+
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: 1.15.7
```

This can be set to latest as it is a library
```suggestion
    tag: latest
```

---

fixed